lib/main_common.pm

```diff
@@ -884,6 +886,10 @@ sub is_mediacheck {
     return get_var('MEDIACHECK');
 }
 
+sub is_upgrade_test {
+    return get_var('UPGRADE') || get_var('ONLINE_MIGRATION') || get_var('ZDUP') || get_var('AUTOUPGRADE');
+}
```

Comment:
How about move it to `version_utils.pm` and just use `is_upgrade` as function name?

Reply:
Yes, sounds good.

Reply:
Yes, I will update according to your comment.
Thanks for the good suggestion.